saas/fsm/renewal_reminder/fsm-renewal_reminder.pt

```diff
@@ -0,0 +1,204 @@
+name "SaaS Manager - Renewal Reminder"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will raise an alert when Flexera SaaS Manager identifies applications whose contract expiration date is approaching. See the [README](https://github.com/rightscale/policy_templates/tree/master/fsm/unsanctioned_spend/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
product doesn't want to use alert. instead change **raise an alert** to **create an incident.**